### PR TITLE
Fixing the whole thing

### PR DESCRIPTION
**Pull request by PavelVanecek**
My last PR actually broke the extension. This PR fixes it again.

The Chrome piece was working the whole time because @dragouf did not publish new version. Firefox one (that I published) was broken from the beginning.

Sorry :/

### CODE REVIEW

File: extension/chrome/src/js/stash_page.js [hunk 1]
```diff
@@ -1996,7 +1996,7 @@
 		}
 	});
 
-	define('bitbucket-plugin/init', [ 'jquery' ], extensionInit);
+	require([ 'jquery' ], extensionInit);
```
Comment (PavelVanecek) [inline extension/chrome/src/js/stash_page.js]: This is why the extension was not working at all: https://addons.mozilla.org/en-US/firefox/addon/bitbucket-server-reviewers/reviews/

**Comment by PavelVanecek** [comment]
@ra100 @Aprillion

File: extension/chrome/src/js/injector_loader.js [hunk 2]
```diff
@@ -3,11 +3,11 @@ const extensionId = chrome.runtime.id || 'stashFF';
 
 function injectEngine(){
 	const manifest = chrome.runtime.getManifest();
-	createInlineScript("var stashRGEVersion = '" + manifest.version + "'; var chromeExtId='" + extensionId + "'; stashIcon='"+chrome.extension.getURL('img/stash128.png')+"';");
+	createInlineScript(`var stashRGEVersion = '${manifest.version}'; var chromeExtId='${extensionId}'; stashIcon='${chrome.extension.getURL('img/stash128.png')}';`);
```
Comment (Aprillion) [inline extension/chrome/src/js/injector_loader.js]: how is this string eval better from just running `window.stashRGEVersion  = ...` plz? is webextension's `window` separated from the page?

File: README.md [hunk 3]
```diff
@@ -98,4 +98,18 @@ After that when you will go to pull request creation page or update page a dropd
 
 **Note**: the extension will make a bitbucket server api request to find reviewers. It will simply send the string you added in the reviewers array as search term. Normally if you add email or username as recommanded API should return only one user. You can also enter a name but in this case if the API return more than one user, only the first one will be added.
 
+### Development troubleshooting
 
+Firefox in development mode logs **Error: The storage API will not work with a temporary addon ID. Please add an explicit addon ID to your manifest. For more information see https://bugzil.la/1323228.**
```
Comment (Aprillion) [inline README.md]: +1

File: extension/chrome/src/js/injector_loader.js (hunk 2 above)
Comment (PavelVanecek) [reply extension/chrome/src/js/injector_loader.js]: Ummm ... probably. I did not change this logic other than eslint autofix. Maybe @dragouf knows?

Comment (Aprillion) [reply extension/chrome/src/js/injector_loader.js]: oh, just eslint autofix. ok.

**Pull request merged**